src/OrchardCore.Modules/OrchardCore.Lists/AdminNodes/ListsAdminNodeDriver.cs

```diff
@@ -58,7 +74,19 @@ private async Task<List<SelectListItem>> GetContentTypesSelectListAsync()
         return (await _contentDefinitionManager.ListTypeDefinitionsAsync())
             .Where(ctd => ctd.Parts.Any(p => p.PartDefinition.Name.Equals(nameof(ListPart), StringComparison.OrdinalIgnoreCase)))
             .OrderBy(ctd => ctd.DisplayName)
-            .Select(ctd => new SelectListItem { Value = ctd.Name, Text = ctd.DisplayName })
+            .Select(ctd => new SelectListItem(ctd.DisplayName, ctd.Name))
             .ToList();
     }
+
+    private async Task<bool> IsValidContentTypeAsync(string contentType)
+    {
+        var definition = await _contentDefinitionManager.GetTypeDefinitionAsync(contentType);
+
+        if (definition is null)
+        {
+            return false;
+        }
+
+        return definition.Parts.Any(p => p.PartDefinition.Name.Equals(nameof(ListPart), StringComparison.OrdinalIgnoreCase));
```

Comment:
We should probably also warn that the Content Type should have a ListPart in case someone removes it from Admin UI.